machine-learning/tutorials/IrisFlowerClustering/Program.cs

```diff
@@ -25,11 +25,11 @@ static void Main(string[] args)
             // </SnippetCreateContext>
 
             // <SnippetSetupTextLoader>
```

Comment:
Please remove SnippetSetupTextLoader. Otherwise looks good and I'll merge after the change.

Reply:
@JRAlexander  Did the changes.